### PR TITLE
Facebook login

### PR DESCRIPTION
**Pull request by gzamudio**
### Summary
Facebook login

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+
+#### Identity
+
+In order to add authentication in our application, we use Identity. For this, we must add the Identity Package:
+
+```bash
+    dotnet add package Microsoft.AspNetCore.Identity.EntityFrameworkCore --version 2.0.0
+```
+
+Now we have to configure it. First, we have to create a User model that inherits from IdentityUser, for example:
```
Comment (gussiciliano) [inline README.md]: Then, we need to configure it. First, we have to create a User model that inherits from IdentityUser, for example:

File: README.md [hunk 2]
```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+
+```bash
+    public class ApplicationUser : IdentityUser 
+    {        
+        public virtual ICollection<IdentityRole> Roles { get; set; }
+    }
+```
+Then, in ```ConfigureServices``` method on ```Startup.cs```, we have to add:
```
Comment (gussiciliano) [inline README.md]: Then, in ```ConfigureServices``` method on ```Startup.cs```, we need to add:

File: README.md [hunk 3]
```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+Also in this method you can set de login and access denied path:
+
+```bash
+services.ConfigureApplicationCookie(options => {
+        options.LoginPath = "/Account/Login";
+        options.AccessDeniedPath = "/Account/AccessDenied";
+    });
+```
+
+And in ```Configure``` method:
```
Comment (gussiciliano) [inline README.md]: Finally, in ...

File: README.md [hunk 4]
```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+```bash
+    app.UseAuthentication();
+```
+
+#### External login
+
+We can also configure external logins with Google, Facebook. OpenId and more. 
+
+## Google
```
Comment (gussiciliano) [inline README.md]: remove this title

File: README.md [hunk 5]
```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+
```
Comment (gussiciliano) [inline README.md]: remove this space

File: README.md [hunk 6]
```diff
@@ -98,6 +98,88 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+```
+Then, in ```ConfigureServices``` method on ```Startup.cs```, we need to add:
+
+```bash
+    services.AddIdentity<ApplicationUser, IdentityRole>()
+        .AddEntityFrameworkStores<ApplicationDbContext>()
+        .AddDefaultTokenProviders();
+```
+
+Also in this method you can set de login and access denied path:
```
Comment (truccomarcos) [inline README.md]: the*

File: README.md [hunk 7]
```diff
@@ -98,6 +98,88 @@ This will generate the controler for that model, along with Create, Edit, Delete
...
+
+Finally, in ```Configure``` method:
+```bash
+    app.UseAuthentication();
+```
+
+#### External login
+
+We can also configure external logins with Google, Facebook. OpenId and more. 
+For example, to add Google Authentication, we have to add de package:
```
Comment (truccomarcos) [inline README.md]: the*

**Pull request merged**